src/pages/distribution.vue

```diff
@@ -34,60 +31,23 @@ const agreeHref = computed(() => {
       </Checkbox>
     </div>
     <div class="flex flex-wrap gap-2 mt-2">
-      <article class="card">
-        <h2>UTAU音源</h2>
-        <img :srcset="utauImage" />
+      <article class="card"  v-for="(contents,dist_name) in mainfile">
+        <h2>{{ dist_name }}</h2>
+        <img :srcset="base + 'distribution/' + contents['image'] + '.webp'" />
```

Comment:
1枚だけならsrcの方が良さそう。
```suggestion
        <img :src="`${base}distribution/${contents['image']}.webp`" />
```
あとTemplate string literalの方が綺麗に書けそう。